src/main/java/be/syntra/devshop/DevshopFront/models/StatusNotification.java

```diff
@@ -9,6 +9,7 @@
 
     PASSWORD_NO_MATCH("The provided passwords don't match!"),
     REGISTER_FAIL("Registering new user failed. Try a different emailaddress"),
+    UPDATED("The update was succesful!"),
```

Comment:
mind the spelling of 'successful' ;-)

Reply:
https://www.spellchecker.net/misspellings/succesfull